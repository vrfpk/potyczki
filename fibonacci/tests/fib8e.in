7670886141691093
