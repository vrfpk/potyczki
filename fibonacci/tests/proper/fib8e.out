22499999999990227
